src/lib/rest/rest.cpp

```diff
@@ -861,14 +863,12 @@ std::string defaultServicePath(const char* url, const char* method)
    */
   if (strcasestr(url, "contextAvailabilitySubscriptions") != NULL)      return DEFAULT_SERVICE_PATH_RECURSIVE;
   if (strcasestr(url, "contextSubscriptions") != NULL)                  return DEFAULT_SERVICE_PATH_RECURSIVE;
-  if (strcasecmp(method, "POST") == 0)                                  return DEFAULT_SERVICE_PATH;
-  if (strcasecmp(method, "PUT") == 0)                                   return DEFAULT_SERVICE_PATH;
-  if (strcasecmp(method, "DELETE") == 0)                                return DEFAULT_SERVICE_PATH;
-  if (strcasecmp(method, "GET") == 0)                                   return DEFAULT_SERVICE_PATH_RECURSIVE;
-  if (strcasecmp(method, "PATCH") == 0)                                 return DEFAULT_SERVICE_PATH;
 
-  std::string details = std::string("cannot find default service path for: (") + method + " " + url + ") - BAD VERB?";
```

Comment:
That line and the next, the call to alarmMgr.badInput.

This call comes before the 'good one', which means it is taken instead and the error message is not as good, that's why I removed it